core/src/main/scala/org/apache/spark/scheduler/SparkListener.scala

```diff
@@ -296,6 +296,17 @@ class StatsReportListener extends SparkListener with Logging {
     taskInfoMetrics.clear()
   }
 
+  private def getStatusDetail(info: StageInfo): String = {
+    val failureReason = info.failureReason.map("(" + _ + ")").getOrElse("")
```

Comment:
It would produce an empty () when it didn't exist or that was the intent I was just following.